src/classes/window/presenter.vala

```diff
@@ -1011,13 +1011,13 @@ namespace pdfpc.Window {
             this.current_view.display(current_slide_number);
 
             var next_view_user_slide = current_user_slide_number;
-            if (!Options.final_slide_overlay ||
-                this.metadata.is_user_slide(current_slide_number)) {
+            bool show_final_slide_of_current_overlay = Options.final_slide_overlay && !this.metadata.is_user_slide(current_slide_number);
```

Comment:
Please avoid long lines exceeding 80 chars, here and elsewhere.

Reply:
I've added additional line breaks for both lines exceeding 80 chars.